probatus/metric_volatility/metric.py

```diff
@@ -110,7 +110,7 @@ def get_metric(
     clf = clf.fit(X_train, y_train)
 
     results_columns = ["metric_name", "train_score", "test_score", "delta_score"]
-    results = pd.DataFrame([], columns=results_columns)
```

Comment:
Without this change, the dtypes of the columns returned by this function would be `object` instead of `float64` in some versions of pandas.

Creating a completely empty data frame avoid this issue from happening without changing functionalities.